estate/views/estate_property_views.xml

```diff
@@ -17,6 +25,28 @@
     </field>
   </record>
 
+  <record id="estate_property_type_tree" model="ir.ui.view">
+    <field name="name">estate.property.type.tree</field>
+    <field name="model">estate.property.type</field>
+    <field name="arch" type="xml">
+      <tree string="Property Types">
+        <field name="sequence" widget="handle"/>
+        <field name="name"/>
+      </tree>
+    </field>
+  </record>
```

Comment:
Forgot to mention this before, we use a 4 columns indent in XML, python and javascript.